src/redux/categories/categories.js

```diff
@@ -0,0 +1,22 @@
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+// action
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+const initialCategoriesState = {
+  categories: [],
+  isChecking: false,
+  message: '',
+};
+const categoriesSlice = (state = initialCategoriesState, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return { ...state, isChecking: true, message: 'Under construction' };
+    default:
+      return state;
+  }
+};
+
+export default categoriesSlice.reducer;
```

Comment:
- [ ] Great job implementing the changes requested by the previous reviewer, But you missed updating the reducer export. Please fix the categories export as it should be `export default categoriesSlice;`.